.github/workflows/android.yml

```diff
@@ -3,13 +3,13 @@ name: Android Build
 # Controls when the workflow will run
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
-  # push:
-  #   branches: 
-  #   - main
+   push:
+     branches: 
+     - upgradeProjectWithLatest
```

Comment:
We cannot change this file for each feature branch. I think we should set it up for dev branch.